contracts/test/integration/ArbitrumTestVoltSystemOracle.t.sol

```diff
@@ -335,11 +335,12 @@ contract ArbitrumTestVoltSystemOracle is DSTest {
         uint256 startingFeeDai = daiPSM.mintFeeBasisPoints();
         uint256 startingFeeUsdc = usdcPSM.mintFeeBasisPoints();
 
-        if (startingFeeDai == 0 && startingFeeUsdc == 0) {
+        /// if starting fee is 5 bips, no need to run this test as upgrade has been applied
+        if (startingFeeDai == 5 && startingFeeUsdc == 5) {
             return;
         }
 
-        vm.startPrank(0x980A05105a53eCa7745DA40DF1AdE6674fc73eD5);
+        vm.startPrank(ArbitrumAddresses.TIMELOCK);
```

Comment:
Nice! There are a lot more places where I think it'd make sense to update this from magic string -> constant https://video.russell.work/GZtwvO

Reply:
This should be another PR, don't want to further clutter this one